apps/web/src/pages/index.tsx

```diff
@@ -164,30 +174,47 @@ const Query = z.object({
 
 export const getServerSideProps = (ctx: GetServerSidePropsContext) => {
   const query = Query.parse(ctx.query);
+  const locale = ctx.locale;
+
+  const t = getTranslations(locale);
+  const isEnglish = locale === 'en' || locale === 'en-US';
+
+  console.log(
+    locale,
+    `../../packages/engine/errors/${isEnglish ? '' : locale}`,
+  );
+
   if (query.error) {
     const decodedError = decompressFromEncodedURIComponent(query.error)!;
     return {
       props: {
         errors: parseErrors(decodedError)
           .reverse()
-          .map((error): ErrorInfo => {
-            return {
-              ...error,
-              improvedError: getImprovedMessageFromMarkdown(
-                path.resolve(process.cwd(), '../../packages/engine/errors'),
-                error.code,
-                error.parseInfo.items,
-              ),
-            };
-          }),
+          .map(
+            (error: any): ErrorInfo => {
+              console.log(error);
+
+              return {
+                ...error,
+                improvedError: getImprovedMessageFromMarkdown(
+                  path.resolve(
+                    process.cwd(),
+                    `../../packages/engine/errors/${isEnglish ? '' : locale}`,
```

Comment:
We should move existing errors into an `en` folder

Reply:
This doesn't break the code for the vs code extension?

Reply:
No, as long as we also _change_ the code for the vscode extension.

Reply:
Fixed on https://github.com/mattpocock/ts-error-translator/pull/92/commits/47c7bb29d032cd080221016d930ae40c62e053a6